Week-7/Ex6_4.java

```diff
@@ -0,0 +1,43 @@
+
+
+import java.util.*;
+public class Ex6_4 {
+	
+	public static int buyAndSellStock(int arr2[]) {
+		int profit = 0;
+		int c1 = 0;
+		int c2 = 1;
+		for(int i = 0; i<arr2.length-1 	; i++) {
+			if(arr2[c1]> arr2[c2]) {
```

Comment:
why dont you move the difference of arr2[c1] and arr2[c2] to a const. You are using it twice